concourse/model/base.py

```diff
@@ -25,6 +25,17 @@
 )
 
 
+class EnumWithDocumentation(Enum):
+    def __new__(cls, value, docstring=None):
```

Comment:
would it not suffice to overwrite the `__init__` method?

Reply:
No, because I am changing the way value assignment works for this enum, `__new__` is the way to go as per doc.

Reply:
you are right; I did not read through the documentation thorough enough yesterday, sry for that